test-cases/features/scale-cluster.yaml

```diff
@@ -0,0 +1,22 @@
+test_duration: 1440
+
+cassandra_stress_threads: 500
+cassandra_stress_population_size: 10000000
+space_node_threshold: 100
+
+n_db_nodes: 450
+cluster_target_size: 500
+add_node_cnt: 1
+seeds_num: 5
+seeds_first: 'true'
```

Comment:
in Yaml everything that is put in single quotation mark is string. As I understand `seed_first` is boolean, so no need to put quotation marks

Reply:
It will be changed after the #1336 merge